GSA/digitalgov.gov
# HCD #4: Refactor Previous/Next Buttons

## Summary

- Extracts individual buttons into a partial to simplify code logic. 
- Removes support for unnecessary third level of navigation. 
- Added arrow icons and text of previous/next page to improve clarity.

### Preview

[Link to Preview](https://federalist-466b7d92-5da1-4208-974f-d61fd4348571.sites.pages.cloud.gov/preview/gsa/digitalgov.gov/jtm-refactor-prev-next/)



### Solution
<img width="977" alt="Screenshot 2023-08-07 at 9 09 41 AM" src="https://github.com/GSA/digitalgov.gov/assets/56370370/f80039e1-5212-49f7-bc90-913f51782e65">

### How To Test

Ensure there are no regressions on any of the existing guides pages, and that the buttons function as intended on guides with multiple levels of navigation. (You will need to create a new .yml file in the `guidedata` folder to test this, along with the associated markdown files. See the HCD guide[ .yml files](https://github.com/GSA/digitalgov.gov/blob/jtm-create-hcd-guides/data/guidenav/hcd-design-concepts.yml) and [markdown](https://github.com/GSA/digitalgov.gov/tree/jtm-create-hcd-guides/content/guides/hcd/design-concepts) for an example.) 

---

### Dev Checklist
